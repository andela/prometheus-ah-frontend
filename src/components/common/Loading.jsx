import React from 'react';
import loader from '../../images/loader.gif';

const Loading = () => (
  <div className="loading-center">
    <img src={loader} className="text" alt="Loading..." />
  </div>
);

export default Loading;
