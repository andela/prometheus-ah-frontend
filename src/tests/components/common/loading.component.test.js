import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../../components/common/Loading';

test('should render LoadingPage correctly', () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper).toMatchSnapshot();
});
