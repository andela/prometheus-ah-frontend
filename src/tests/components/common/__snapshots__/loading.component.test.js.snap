// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="loading-center text-center"
>
  <img
    alt="Loading..."
    className="text"
    src={Object {}}
  />
</div>
`;
